modules/rosa-hcp-sts-creating-a-cluster-external-auth-provider-cli.adoc

```diff
@@ -0,0 +1,128 @@
+// Module included in the following assemblies:
+//
+// * rosa_hcp/rosa-hcp-sts-creating-a-cluster-quickly.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="rosa-hcp-sts-creating-a-cluster-external-auth-provider-cli_{context}"]
+= Creating an external authentication provider
+:source-highlighter: pygments
+:pygments-style: emacs
+:icons: font
+ 
+After you have created a {hcp-title} cluster with the enabled option for external authentication providers, you must create a provider using the ROSA CLI.
+
+This configuration uses Microsoft Entra ID to authenticate your credentials. The Entra ID values are:
+
+[cols="1,1", options="header"]
+|===
+
+|CLI Flag
+|Expected value
+
+|`--cluster` 
+|This string is the name or the ID of your cluster
+
+|`--name`
+|This string is the name you set for your external authentication provider.
+
+|`--issuer-audiences` 
+|This common-separated list of strings is your client ID. In Microsoft Entra ID, this is called the audience ID.
+
+|`--issuer-url`
+|This string is the URL of the token issuer
+
+|`--claim-mapping-username-claim` 
+|This string is the name of the claim that should be used to construct user names for the cluster identity.
+
+|`--claim-mapping-groups-claim`
+| This string describes rules on how to transform information from an ID token into a cluster identity.
+
+|===
+
+.Procedure
+
+. Use one of the following commands to create your external authentication provider:
+
+* To use the interactive command interface, run the following commands:
++
+[source,terminal]
+----
+$ rosa create external-auth-provider -c <cluster_name>
+----
++
+This command starts an interactive CLI process:
++
+--
+[source,terminal]
+----
+I: Enabling interactive mode
+? Name: <.>
+? Issuer audiences: <.>
+? The serving url of the token issuer: <.>
+? CA file path (optional): <.>
+? Claim mapping username: <.>
+? Claim mapping groups: <.>
+? Claim validation rule (optional): <.>
+? Console client id (optional): <.>
+----
+<.> The name of your external authentication provider. This name should be a lower-case with numbers and dashes.
+<.> The issuer audiences field requires you to provide the audience IDs that this authentication provider issues tokens for.
+<.> This field uses the issuer's URL that serves the token.
+<.> Optional: This field takes the path of a certificate file to use when making requests.
+<.> The name of the claim that is used to construct the user names for cluster identity, such as using `email`.
+<.> This field specifies how to transform the ID token into a cluster identity, such as using `groups`.
+<.> Optional: These are rules that help validate token claims which authenticate your users. This field should be formatted as `<claim:<required_value>`.
+<.> Optional: This field is the application or client ID that your app registration uses for the console.
```

Comment:
Is there a reason why you shifted from using hard-coded numbers <1> to the wildcard <.>?

Reply:
```suggestion
<.> Optional: The application or client ID that your app registration uses for the console.
```